mdl/src/main/cft/mdlHerdRds.yml

```diff
@@ -151,7 +151,7 @@ Resources:
         rds.force_ssl: '1'
   PostgresDB:
     Type: 'AWS::RDS::DBInstance'
-    DeletionPolicy: Snapshot
+    DeletionPolicy: Delete
```

Comment:
Why would you delete the final snapshot?

Reply:
I was thinking to add this as a cleanup step which deletes SNAPSHOT after RDS deletion, if this is not an usual case, I can revert this back

Reply:
Yes please, revert this.